x/concentrated-liquidity/incentives.go

```diff
@@ -42,6 +42,18 @@ func getUptimeTrackerValues(uptimeTrackers []model.UptimeTracker) []sdk.DecCoins
 	return trackerValues
 }
 
+func (k Keeper) GetAccumulators(ctx sdk.Context, poolId uint64) (accum.AccumulatorObject, []accum.AccumulatorObject, error) {
```

Comment:
non-blocking nit: consider having it in `swaps.go` as this helper is only used there and naming it `getAccumulatorsForSwaps`

Having it in the incentives files when it also returns spread accums feels odd to me

Reply:
Makes sense, applied!